File: GTTS/INL.TrainingService.Models/GetTrainingEvents_Item.cs
using System;
using System.Collections.Generic;

namespace INL.TrainingService.Models
{
    public class GetTrainingEvents_Item : IGetTrainingEvents_Item
    {
        public long TrainingEventID { get; set; }
        public string Name { get; set; }
        public string NameInLocalLang { get; set; }
        public string TrainingEventType { get; set; }
        public int? ProgramID { get; set; }
        public string BusinessUnitAcronym { get; set; }
        public string BusinessUnit { get; set; }
        public int? OrganizerAppUserID { get; set; }
        public string Organizer { get; set; }
        public int? TrainingEventStatusID { get; set; }
        public string TrainingEventStatus { get; set; }
        public DateTime? EventStartDate { get; set; }
        public DateTime? EventEndDate { get; set; }
        public int? ModifiedByAppUserID { get; set; }
        public string ModifiedByAppUser { get; set; }
        public DateTime? ModifiedDate { get; set; }
        public DateTime? CreatedDate { get; set; }

        public List<GetTrainingEventLocation_Item> TrainingEventLocations { get; set; }
        public List<GetTrainingEventKeyActivity_Item> TrainingEventKeyActivities { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-training-events_item.ts


import { GetTrainingEventLocation_Item } from './get-training-event-location_item';
import { GetTrainingEventKeyActivity_Item } from './get-training-event-key-activity_item';

export class GetTrainingEvents_Item  {
  
	public TrainingEventID: number = 0;
	public Name: string = "";
	public NameInLocalLang: string = "";
	public TrainingEventType: string = "";
	public ProgramID?: number;
	public BusinessUnitAcronym: string = "";
	public BusinessUnit: string = "";
	public OrganizerAppUserID?: number;
	public Organizer: string = "";
	public TrainingEventStatusID?: number;
	public TrainingEventStatus: string = "";
	public EventStartDate?: Date;
	public EventEndDate?: Date;
	public ModifiedByAppUserID?: number;
	public ModifiedByAppUser: string = "";
	public ModifiedDate?: Date;
	public CreatedDate?: Date;
	public TrainingEventLocations?: GetTrainingEventLocation_Item[];
	public TrainingEventKeyActivities?: GetTrainingEventKeyActivity_Item[];
  
}






